ArgoTests/JSON/JSONFileReader.swift

```diff
@@ -4,7 +4,16 @@ import Runes
 func JSONFromFile(file: String) -> AnyObject? {
   return NSBundle(forClass: JSONFileReader.self).pathForResource(file, ofType: "json")
     >>- { NSData(contentsOfFile: $0) }
-    >>- { NSJSONSerialization.JSONObjectWithData($0, options: NSJSONReadingOptions(0), error: nil) }
+    >>- parseJSON
 }
 
 private class JSONFileReader { }
+
+func parseJSON(data: NSData) -> AnyObject? {
```

Comment:
this is unfortunate ... cocoa apis that have the error parameter like `NSJSONSerialization` will now throw errors so they need to be handled manually